sdk/storage/azure-storage-blobs/samples/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
```

Comment:
nit. Ideally, we might want to have one folder for each sample. Each folder would contain a CMakeList and the source code.

In the future, we want to make then entire folder a copy/paste project with no dependency to the repo.

This works for now and is better than previous approach.